pandas/core/arrays/base.py

```diff
@@ -128,6 +128,30 @@ def _from_sequence(cls, scalars, dtype=None, copy=False):
         """
         raise AbstractMethodError(cls)
 
+    @classmethod
```

Comment:
can you add ``_from_sequence_of_strings`` in the Notes section at the top, next to the other constructors

Reply:
actually this is no required, so put it separately